### PR TITLE
unsafe foods blog number 1

### PR DESCRIPTION
**Pull request by cvint13**
added first blog post for our group

### CODE REVIEW

**Comment by arokem** [comment]
There seem to be two files added in the _posts directory? Maybe you changed the file name and forgot to `git rm` the other one? Either way, please don't use ALL-CAPS in the file-name.

File: _posts/2016-06-30-Unsafe-Foods-Week-2.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+---
+
+<body>
+
+<p>
+The Unsafe Foods project is going to be very exciting and challenging this summer. We get to experiment with advanced text mining practices and attempt to make a predictive model for product recalls from Amazon reviews. As intriguing as this topic is, our goals will not be achieved without a lot of sifting through some muddy data sets and determining what we can accomplish given our time frame and resources.
+</p>
+
+<p>
+For the first week of the Unsafe Foods project, our team had to figure out how to grapple with two separate data sets with very different formats. For the Amazon review data, we are using a structured <a href="http://jmcauley.ucsd.edu/data/amazon/">data set</a> curated and published by Julian McAuley from UCSD. The challenge arose in capturing a workable data set for food recalls from the FDA. Kara Woo managed to find an <a href="https://web.archive.org/web/20150504011324/http://www.fda.gov/DataSets/Recalls/RecallsDataSet.xml">XML version</a> of a history of food recalls with a link to each recall's press release containing each product's unique identifier(s).
```
Comment (arokem) [inline _posts/2016-06-30-Unsafe-Foods-Week-2.md]: Maybe you want to link to Kara's intro post here, when you mention here?

File: _posts/2016-06-30-Unsafe-Foods-Week-2.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+</p>
+
+<p>
+This disconnected data required some creativity in order to bring them together. Each Amazon review was assigned an ASIN, or Amazon Standard Identification Number. Each ASIN corresponds to 1 UPC, or Universal Product Code. Unfortunately, while the Recall XML data contains unique names, the UPCs are listed only when available, and in no particular format within the press releases. We needed to fiure out a way to capture those UPCs to match them against the reviews' ASINs.
```
Comment (arokem) [inline _posts/2016-06-30-Unsafe-Foods-Week-2.md]: => "These disconnected data..."?

File: _posts/2016-06-30-Unsafe-Foods-Week-2.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+</p>
+
+<p>
+The second issue was trickier. We had to research the nature of the UPC, and figure out if there was any way to determine the corresponding 12-digit UPC from the provided 10-digit format. Miki Verma saved the day, though, and she figured out an algorithm to determine the 10 possible 12-digit UPCs from the 10-digit UPC. From there, we can infer that there is only one possible 12-digit UPC among the 10 possibilities. This is because the first digit is based on the type of food, the next 5 digits correspond to the company ID, the next 5 digits are reserved for the company-wide product ID, and the last digit is a check digit to confirm the validity of the UPC. With the 10-digit UPC, the company ID is known, and the product ID is known. Each product has only one possible first digit. Once the correct UPC is passed into the REST API and a corresponding ASIN is returned, the function can discard the rest.
```
Comment (arokem) [inline _posts/2016-06-30-Unsafe-Foods-Week-2.md]: Same here for Miki.

**Comment by arokem** [comment]
Great! Just a couple of small comments on my side. I'll let @valentina-s review and merge.

**Comment by valentina-s** [comment]
Looks great! Just one typo - fiure instead of figure in the same paragraph as the disconnected data.

**Comment by valentina-s** [comment]
@cvint13 
i merged it but there seems to be extra 'fi' in the layout specification, so the blog does not show properly. If you remove those characters should be fine.

**Pull request merged**